### PR TITLE
Updated the frontend, used a loader for page loading and also implemented authenticated sign up without bankend.

### PR DESCRIPTION
**Pull request by Ayush604141**
Updated the frontend, used a loader for page loading and also implemented authenticated sign up without backend.
Special thanks to @ImShubhK for helping me in page loader and authentication.

### CODE REVIEW

**Review by PatelNeelMahesh — CHANGES_REQUESTED** [review]
Ayush looks like you have added .node_modules, you should add it in .env.

**Comment by Ayush604141** [comment]
I have added the .gitignore files in the backend folder for good. I hope the changes will be visible soon.

File: frontend/src/components/templates/Temp1.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./template1.css";
+import HolderImage from "../assets/choose-image.png";
+
+const Temp1 = () => {
+  const handleImageInput = () => {
```
Comment (PatelNeelMahesh) [inline frontend/src/components/templates/Temp1.js:6]: Go through, https://timmousk.com/blog/react-get-element-by-id/ and make the changes accordingly.

**Review by PatelNeelMahesh — APPROVED** [review]
you can merge pull request, after going through the blog.

File: frontend/src/components/templates/Temp1.js (hunk 1 above)
Comment (Ayush604141) [reply frontend/src/components/templates/Temp1.js:6]: I have gone through this and I will use useRef hooks in my next PR Today.

**Comment by PatelNeelMahesh** [comment]
Add description for the PR, 
image
Click on edit and add description.

**Pull request merged**